src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -505,7 +505,15 @@ function enableNewQuestion() {
 
     $newQuestionTable.find(`#rubricAddChoiceLink-${NEW_QUESTION}`).show();
     $newQuestionTable.find(`#rubricAddSubQuestionLink-${NEW_QUESTION}`).show();
-    $newQuestionTable.find(`#rubricWeights-${NEW_QUESTION}`).hide();
+
+    // Prevents the weight input field from not appearing when the instructor
```

Comment:
This sounds a little confusing :sweat_smile: . Something like -

```js
// If instructor had assigned rubric weights before,
// then display the weights row, otherwise hide it.
```

might be better.